netbox/extras/tests/test_api.py

```diff
@@ -14,14 +14,12 @@
 from extras.scripts import BooleanVar, IntegerVar, Script, StringVar
 from utilities.testing import APITestCase, APIViewTestCases
 
-
```

Comment:
Please revert all these arbitrary formatting changes. No changes to this file are needed.